localstack-core/localstack/services/apigateway/next_gen/execute_api/integrations/mock.py

```diff
@@ -63,48 +63,46 @@ def parse_invalid_json(self, body: str) -> dict:
         CDK creates a MOCK OPTIONS route with in valid json. `{statusCode: 200}`
         Aws probably has a custom token parser. We can implement one
         at some point if we have user requests for it"""
+
+        def convert_null_value(value) -> str:
+            if value in ("null", ""):
+                return '""'
+            return value
+
         try:
             statuscode = ""
             matched = re.match(r"^\s*{(.+)}\s*$", body).group(1)
-            splits = [m.strip() for m in matched.split(",")]
+            pairs = [m.strip() for m in matched.split(",")]
             # TODO this is not right, but nested object would otherwise break the parsing
-            kvs = [s.split(":", maxsplit=1) for s in splits]
-            for kv in kvs:
-                assert len(kv) == 2
-                k, v = kv
-                k = k.strip()
-                v = v.strip()
-
-                assert k
-                assert v
-
-                if k == "statusCode":
-                    statuscode = int(v)
+            key_values = [s.split(":", maxsplit=1) for s in pairs if s]
+            for key_value in key_values:
+                assert len(key_value) == 2
+                key, value = [el.strip() for el in key_value]
+                key = convert_null_value(key)
+                value = convert_null_value(value)
```

Comment:
nit: could probably be simplified with:
```suggestion
                key, value = [convert_null_value(el.strip()) for el in key_value]
```

I think we could even put the `strip()` call inside `convert_null_value`.

Reply:
Oops, missed it but we could probably add type hints for `convert_null_value(value: str) -> str:`